### ISSUE TITLE
Parameter initialization problem.

### ISSUE THREAD

**Issue by codeshop715** [bfshi/TOAST#5]
Your work is excellent. I am inspired to follow your progress and undertake some intriguing endeavors. During my experiments, I have come to realize the paramount importance of initializing the top-down attention component. Could you please elaborate on how the initialization process for the feedback path (decoders) and the feature selection part (prompt and top_down_transform) occurs? Were they pre-trained on imagenet1k?Was varloss used in the pre-training process? It would be greatly appreciated if you could provide the pre-training parameters. I eagerly await your response.

**Comment by bfshi** [comment]
Yes the feedback path and the feature selection part are pre-trained (in our paper we called it 'pre-tuned') on ImageNet-1k. Var loss is used during the pre-trained process. The pre-tuned parameters are provided for several models (please see [here](https://github.com/bfshi/TOAST/tree/main/visual_classification#model-zoo))

**Comment by codeshop715** [comment]
Thank you very much for your reply.
